modules/exploits/linux/http/cisco_rv340_lan.rb

```diff
@@ -0,0 +1,163 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Cisco RV34X Series Authentication Bypass and Command Injection',
+        'Description' => %q{
+          This module exploits a sessionid directory traversal authentication bypass, a sessionid
+          improper input validation authentication bypass, and command injection on Cisco RV340 and
+          RV345 Small Business Routers. All vulnerabilities were discovered by Bien Pham
+          at Sea Security.
+
+          This module works on firmware versions 1.0.03.24 and below
+        },
+        'License' => MSF_LICENSE,
+        'Platform' => ['Linux', 'Unix'],
+        'Author' => [
+          'Biem Pham',  # Vulnerability Discoveries
+          'Neterum',    # Metasploit Module
+          'jbaines-r7'  # This metasploit module is heavily inspired from
+                        # cisco_rv_series_authbypass_and_rce.rb
+        ],
+        'DisclosureDate' => '2021-11-02',
+        'Arch' => [ARCH_CMD, ARCH_ARMLE],
+        'References' => [
+          [ 'URL', 'https://blog.security.sea.com/posts/pwn2own-2021-rv340/'], # Possibly down
+          [ 'CVE', '2022-20701'],
+          [ 'CVE', '2022-20705'],
+          [ 'CVE', '2022-20707']
```

Comment:
You might want to add a reference to https://www.zerodayinitiative.com/advisories/ZDI-22-409/ (and maybe a comment saying that this is the auth bypass?)

Reply:
Added reference.